internal/xds/translator/httpfilters.go

```diff
@@ -13,10 +13,55 @@ import (
 	hcmv3 "github.com/envoyproxy/go-control-plane/envoy/extensions/filters/network/http_connection_manager/v3"
 	"github.com/envoyproxy/go-control-plane/pkg/wellknown"
 
+	"github.com/envoyproxy/gateway/internal/xds/filters"
+	"github.com/envoyproxy/gateway/internal/xds/types"
+
 	"github.com/envoyproxy/gateway/internal/ir"
-	xdsfilters "github.com/envoyproxy/gateway/internal/xds/filters"
 )
 
+var httpFilters = [...]httpFilter{&cors{}, &jwt{}, &oidc{}}
```

Comment:
is this needed ? someone will always forget to add this

Reply:
The tests will never pass if someone forget this, but it will be better to just modify one file. I replaced this a register method in each HTTPFilter.